server/src/main/java/org/apache/iotdb/db/qp/physical/crud/LastQueryPlan.java

```diff
@@ -80,16 +77,12 @@ public void setExpression(IExpression expression) throws QueryProcessException {
     if (isValidExpression(expression)) {
       super.setExpression(expression);
     } else {
-      throw new QueryProcessException("Only '>' and '>=' are supported in LAST query");
+      throw new QueryProcessException("Only time filters are supported in LAST query");
     }
   }
 
   // Only > and >= are supported in time filter
```

Comment:
Change this?

Reply:
Good catch! I have fixed the comment and updated the documents as @ericpai  suggested.